vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/TouchSteps.java

```diff
@@ -134,26 +135,38 @@ private Supplier<Rectangle> createSwipeArea()
         return () -> new Rectangle(new Point(0, 0), genericWebDriverManager.getSize());
     }
 
-    private void adjustVerticalPosition(WebElement element, Rectangle swipeArea, Duration swipeDuration)
+    private void adjustElementPosition(WebElement element, Rectangle swipeArea, Duration swipeDuration,
```

Comment:
I propose to introduce some class which will wrap area and direction and contain the logic of swipeArea size calculation, indents calculation etc

Reply:
@valfirst What do you think about introducing new class?